Tensile/Common.py

```diff
@@ -347,6 +347,11 @@
     # GSU applies only to the unroll summation dimension
     "GlobalSplitU":               list(range(1, 1024+1)),
 
+    # choose how to do GlobalSplitU
+    # 1: use atomic operation to accumlate on one buffer
+    # 2: each GSU group write to each own buffer and accumulate by another kernel
+    "GlobalSplitUAlgorithm":      [1, 2],
```

Comment:
+0 = no GSU?

Reply:
I'm not opposed to using numerical IDs to distinguish an algorithm, however I might suggest updating the implementation to use an enum wrapper class for the ID.
To a maintainer, globalAccumulation: GlobalAccumulateAlgo.MyAlgo may be a bit more verbose, but is more explicit and refactor-safe than globalAccumulation: 0, where meaning of 0 must be looked up, and potentially meaning of 0 could change in the future.

Reply:
no GSU is decided by GlobalSplitU.
GlobalSplitUAlgorithm is a sub parameter of GlobalSplitU.
It only take effect when GlobalSplitU != 1

Reply:
can we use "string" as a key for GlobalSplitUAlgorithm.
Enum is not good for yaml file.
I upload a commit to use string to replace int for GlobalSplitUAlgorithm.

Reply:
Hi @cgmillette,
Does that work for you??